### DISCUSSION TITLE
Windows 11 Cracking Down on Unsupported Upgrades

### DISCUSSION THREAD

**Discussion by Lozt247** [AveYo/MediaCreationTool.bat#369] [General]
Just wondering if this will affect us.

I hope not.

Thank you for all your work

https://www.xda-developers.com/microsoft-cracking-down-dodging-windows-11-system-requirements/?utm_campaign=trueanthem&utm_medium=social&utm_source=facebook&fbclid=IwZXh0bgNhZW0CMTEAAR0h2tXt93fEkt5NKVrrXQphi0OCjCxzVoksDqEs0XUQcYIv8njTfK6pc4g_aem_LSp2Td6OZHVkREl8Cbgphg

**Comment by nondetect** [comment]
It's one of several ways using by this script

**Reply by Lozt247** [reply]
Hoping it will keep working 🤙🤞
